bin/process.sh

```diff
@@ -6,10 +6,15 @@ shift $[$#-2]
 
 input=$1
 output=$2
-# target size in KB
-THUMBNAIL_SIZE=${THUMBNAIL_SIZE:-300}
+
+THUMBNAIL_SIZE=${THUMBNAIL_SIZE:-300} # target size in KB
 TILER_BASE_URL=${TILER_BASE_URL:-http://tiles.openaerialmap.org}
 
+# Although TMPDIR is usually set by the OS, it doesn't seem to be getting passed through
+# Docker and Node's `spawn()`, so no harm done in just defaulting to `/tmp` in such cases.
+# There may be clues here as the underlying issue: https://github.com/npm/npm/issues/4531
+TMPDIR={TMPDIR:-/tmp}
```

Comment:
We shouldn't actually use `/tmp` here, as it will often be `tmpfs` (memory-backed), and as-such, subject to premature purging, particularly with very large files. (@iandees and I saw files disappear out from under us under these circumstances.)

Maybe `$HOME/tmp`, created if necessary?